frontend/src/app/App.tsx

```diff
@@ -42,6 +43,8 @@ const App: React.FC = () => {
     loadError: fetchConfigError,
   } = useApplicationSettings();
 
+  const [storageClasses] = useStorageClasses();
```

Comment:
@alexcreasy beautiful, now it's in line with main, and it's exactly how I tried and modified it a few hours ago locally, neat. Looks good to me.